src/rpc/rawtransaction.cpp

```diff
@@ -761,6 +761,7 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
 {
     RPCHelpMan{"sendrawtransaction",
                 "\nSubmits raw transaction (serialized, hex-encoded) to local node and network.\n"
+                "Note that using this for manual rebroadcast may represent a privacy leak.\n"
```

Comment:
Maybe explain why? (The tx will be sent unconditionally to all peers)

Reply:
I was trying to make the smallest possible change but agree with you. Done, updated commit message and PR description.